VMBackup/main/Utils/DiskUtil.py

```diff
@@ -203,7 +203,7 @@ def get_device_items(self, dev_path):
                      lsblk_path = str(out_which_output)
                      is_lsblk_path_wrong, out_lsblk_output, error_msg = self.get_lsblk_pairs_output(lsblk_path, dev_path)
             # if error_msg contains "invalid option", then get device_items using method get_device_items_from_lsblk_list
-            if (error_msg is not None and error_msg.strip() != "" and 'invalid option' in error_msg):
+            if (error_msg is not None and error_msg.strip() != "" and ('invalid option' in error_msg or 'P' in error_msg)):
```

Comment:
in non-English OS, 'invalid option' is not matching, so this change

Reply:
what is P ?

Reply:
P from '-P' option not present in lsblk… everything else was in chinese

Reply:
please add the same as comment in the code.
Are we sure that "P" is always available in every locale ?

Reply:
will add the comment. How to be sure of if P is available in all locales ? Will do incremental change in fuiture if this does not suffice.

Reply:
Added